server/service/src/invoice/outbound_shipment/update_name/generate.rs

```diff
@@ -0,0 +1,64 @@
+use repository::{
+    EqualFilter, InvoiceLine, InvoiceLineFilter, InvoiceLineRepository, InvoiceLineRow, Name,
+};
+use repository::{InvoiceRow, StorageConnection};
+use util::uuid::uuid;
+
+use super::{UpdateOutboundShipmentName, UpdateOutboundShipmentNameError};
+
+pub struct GenerateResult {
```

Comment:
Batches should stay the same since everything from the old invoice/invoice line has been copied over to the new invoice, so have not returned anything to adjust stock levels.